tests/cluster/tests/04-resharding.tcl

```diff
@@ -84,7 +84,7 @@ test "Cluster consistency during live resharding" {
             flush stdout
             set target [dict get [get_myself [randomInt 5]] id]
             set tribpid [lindex [exec \
-                ../../../src/valkey-cli --cluster reshard \
+                ../../../src/valkey-cli -c --cluster reshard \
```

Comment:
This makes me think that if `--cluster` is provided, `valkey-cli` can infer that it's in cluster mode.